sqswatcher/plugins/slurm.py

```diff
@@ -61,7 +61,7 @@ def _restart_master_node():
         raise
 
 
-@retry(stop_max_attempt_number=3, wait_fixed=10000)
+@retry(stop_max_attempt_number=3, wait_fixed=2000)
```

Comment:
is 2000 enough?

Reply:
2000 is the wait time for this fast retry at function level while the longer retry wait time is given when re-queuing the message